src/main/java/com/jcabi/http/wire/AbstractHeaderBasedCachingWire.java

```diff
@@ -0,0 +1,218 @@
+/**
+ * Copyright (c) 2011-2015, jcabi.com
```

Comment:
At the moment check is configured for 2015:
[ERROR] /src/main/java/com/jcabi/http/wire/ETagCachingWire.java[2]: Line does not match expected header line of ' * Copyright (c) 2011-2015, jcabi.com'. (HeaderCheck)